builtin/client/register.lua

```diff
@@ -61,5 +61,6 @@ core.registered_on_sending_chat_messages, core.register_on_sending_chat_messages
 core.registered_on_death, core.register_on_death = make_registration()
 core.registered_on_hp_modification, core.register_on_hp_modification = make_registration()
 core.registered_on_damage_taken, core.register_on_damage_taken = make_registration()
+core.registered_on_formspec_input, core.register_on_formspec_input = make_registration()
```

Comment:
That name doesn't really make sense for client-sided mods as there is only one player and the fields aren't sent anywhere.